services/cartItem-services.js

```diff
@@ -0,0 +1,27 @@
+const {
+    selectCartItemsByCartItemId,
+    selectCartItemsByCartId
+} = require('../repositories/cartItems-repository');
+
+const mapToModel = (cartItem) => ({
+    cartItemId: cartItem['cartItem_id'],
+    item: cartItem['item'],
+    quantity: cartItem['quantity']
+});
+
+const getCartItemsByCartItemId = (cartItemId) => {
```

Comment:
`getCartItemByCartItemId`

we want a single resource by it's unique identifier